src/content/troubleshooting/_index.en.md

```diff
@@ -153,7 +153,7 @@ Spec:
   Ports:                    <nil>
   Session Affinity:
   Session Affinity Config:  <nil>
-  Type:                     SuperclusterIP
+  Type:                     ClusterSetIP
```

Comment:
This is pending confirmation from the SIG wrt. the MCS API.

Reply:
The change has been confirmed, see https://github.com/JeremyOT/mcs-api/commit/b44bc782c2c1af504d4e43b4f844d7ce004db231